### ISSUE TITLE
Tray icon works, but no main interface when using discrete GPU

### ISSUE THREAD

**Issue by laurento** [flathub/com.valvesoftware.Steam#1163]
**Distribution name and version where applicable**
Debian testing

**Flatpak info**
```
Flatpak 1.14.4

$ flatpak --gl-drivers
default
host
```

**Problem description**
Since a month or so, Steam GUI doesn't appear on screen anymore when I lunch the app using "Discrete Graphics Card". However, the tray icon works as expected.

Everything works fine if I start Steam using "Integrated Graphics Card" instead.

**Does this issue reproduce with native Steam**
Have not tested

**Comment by MisterSheeple** [comment]
Does it get stuck on "Loading user data" for you?

**Comment by laurento** [comment]
Nope, after showing "loading user data" that window disappears and the (working) tray icon appears as expected.

**Comment by rybak** [comment]
This sounds like #1168 and/or https://github.com/ValveSoftware/steam-for-linux/issues/9383 to me.
